packages/rocketchat-crowd/server/crowd.js

```diff
@@ -224,6 +281,11 @@ Accounts.registerLoginHandler('crowd', function(loginRequest) {
 		const crowd = new CROWD();
 		const user = crowd.authenticate(loginRequest.username, loginRequest.crowdPassword);
 
+		if (!user) {
+			logger.debug(`User ${ loginRequest.username } is not a valid crowd user, falling back`);
+			return fallbackDefaultAccountSystem(this, loginRequest.username, loginRequest.crowdPassword);
```

Comment:
Should have a setting to enable/disable this behavior. Some companies would like to block auth if user was removed from Crowd

Reply:
Correct. I changed the behaviour to check if the user has been a crowd user -> then he/she is blocked. If it has been a local user, the fallback method is used.

I would return a 
```
return new Meteor.Error('not-authorized', 'User is not authorized by crowd');
```
in the first case. Is that the correct way here, or should something else be returned to create the correct behaviour in the frontend (like 'you are not authorized') @rodrigok ?